### PR TITLE
Fix izzzj json

### PR DESCRIPTION
**Pull request by kazarmy**
https://github.com/radareorg/radare2/commit/5e78c0a55251412b48169387cb01e8306459e028#diff-5f23cf851d424110d88a88e94a5b38a4

### CODE REVIEW

File: test/new/db/cmd/cmd_i [hunk 1]
```diff
@@ -3464,6 +3464,27 @@ izzj
...
+NAME=izzzj
+FILE=-
+CMDS=<<EOF
+w abcd\00012345\0efghi\0
+izzzj~{}
+EOF
+EXPECT=<<EOF
+[
+  {
+    "string": "abcd"
```
Comment (radare) [inline test/new/db/cmd/cmd_i]: this output should contain 'offset' or 'vaddr', 'length', and so on

File: libr/core/cmd_info.c [hunk 2]
```diff
@@ -1234,7 +1235,7 @@ static int cmd_info(void *data, const char *input) {
 	}
 done:
 	if (is_array) {
-		r_cons_printf ("}\n");
+		r_cons_printf ("%s\n", is_zz ? "]" : "}");
```
Comment (radare) [inline libr/core/cmd_info.c]: better use the pj api

File: libr/bin/bfile.c [hunk 3]
```diff
@@ -62,7 +62,7 @@ static void print_string(RBinFile *bf, RBinString *string, int raw) {
 				pj_o (pj);
 				pj_ks (pj, "string", string->string);
 				pj_end (pj);
-				io->cb_printf ("%s\n", pj_string (pj));
+				io->cb_printf ("%s%s\n", first ? "" : ",", pj_string (pj));
```
Comment (radare) [inline libr/bin/bfile.c]: using the pj api will avoid the need to add this bool first arg

**Comment by radare** [comment]
please rebase

**Comment by radare** [comment]
pls rebase

**Comment by kazarmy** [comment]
ok rebased

File: test/new/db/cmd/cmd_i (hunk 1 above)
Comment (kazarmy) [reply test/new/db/cmd/cmd_i]: #15759

File: libr/bin/bfile.c (hunk 3 above)
Comment (kazarmy) [reply libr/bin/bfile.c]: #15760

**Pull request merged**